src/app.js

```diff
@@ -28,6 +30,32 @@ export function createApp() {
     app.use(require('koa-logger')());
   }
 
+  app.use(
+    reportTo([
+      {
+        group: 'cspreport',
+        maxAge: 365 * 24 * 60 * 60,
+        endpoints: [{ url: '/__cspreport__' }],
+      },
+    ])
+  );
+  app.use(
+    helmet({
+      contentSecurityPolicy: {
+        directives: {
+          defaultSrc: ["'none'"],
+          frameAncestors: ["'none'"],
+          baseUri: ["'none'"],
+          reportUri: '/__cspreport__',
+          reportTo: 'cspreport',
+        },
+      },
+      hsts: {
+        maxAge: 31536000, // 1year
```

Comment:
I'd prefer this number to be `365 * 24 * 60 * 60 // 1 year`, like above, that way its consistent and not magical.